tests/lib/rules/no-invalid-resource-location.js

```diff
@@ -36,20 +36,26 @@ ruleTesterInstance.run('no-invalid-resource-location', ruleUnderTest, {
   ],
   invalid: [
     {
-      code: 'export const useAuth = ()=>{/* definition */}',
+      code: 'export const dialog = ()=>{/* definition */}',
       errors: [
         {
           message: 'All components and their related styles should be placed in their own directory (.../components/{component-name}/{component-name}.component.tsx)',
         },
+        {
+          message: 'The file-name and the parent directory name should match',
+        },
       ],
       filename: 'frontend/src/modules/common/components/dialog.component.tsx',
     },
     {
-      code: 'export const useAuth = ()=>{/* definition */}',
+      code: 'export const dialogPartial = ()=>{/* definition */}',
       errors: [
         {
           message: 'All partials and their related styles should be placed either in their own directory (.../partials/{partial-name}/{partial-name}.partial.tsx) or alongwith a related component in their directory',
```

Comment:
@udyansharma-01 I'm not sure that this message is correct, partials should always be in separate directory.